packages/schematics/angular/web-worker/index.ts

```diff
@@ -139,8 +139,10 @@ function addSnippet(options: WebWorkerOptions): Rule {
     }
 
     const siblingModules = host.getDir(options.path).subfiles
-      // Find all files that start with the same name, are ts files, and aren't spec files.
-      .filter(f => f.startsWith(options.name) && f.endsWith('.ts') && !f.endsWith('spec.ts'))
+      // Find all files that start with the same name, are ts files,
+      // and aren't spec or module files.
+      .filter(f => f.startsWith(options.name) && f.endsWith('.ts')
+      && !f.endsWith('spec.ts') && !f.endsWith('-.module.ts'))
```

Comment:
This doesn't look right the `-.module.ts` will not match any file that is generated via the CLI.

I think you meant something like
```ts
.filter(f => f.startsWith(options.name) && f.endsWith('.ts')
      && !/\.(module|spec)\.ts$/.test(f))

```